smarts/env/tests/test_social_agent.py

```diff
@@ -19,14 +19,16 @@
 # LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
-import gym
+import gymnasium as gym
 import pytest
 
 from smarts.core.agent_interface import AgentInterface, AgentType
-from smarts.env.hiway_env import HiWayEnv
+from smarts.env.gymnasium.hiway_env_v1 import HiWayEnvV1
+from smarts.env.utils.action_conversion import ActionOptions
+from smarts.env.utils.observation_conversion import ObservationOptions
 
 AGENT_ID = "Agent-007"
-SOCIAL_AGENT_ID = "Alec Trevelyan"
```

Comment:
I see you are getting rid of the traitor. 🤣

Reply:
Because the traitor was not being tested properly ...